backend/tests/test_unknown_resalloc_tickets.py

```diff
@@ -0,0 +1,57 @@
+"""
+Test the ./run/copr-backend-unknown-resalloc-tickets.py script
+"""
+
+import contextlib
+import importlib
+import io
+import subprocess
+from unittest import mock
+
+from testlib import patch_path_run, patch_path_fake_executables, patch_getpwuid
+
+SCRIPT_NAME = 'copr-backend-unknown-resalloc-tickets.py'
+SPEC = importlib.util.spec_from_file_location("spec", "run/" + SCRIPT_NAME)
+SCRIPT = importlib.util.module_from_spec(SPEC)
+SPEC.loader.exec_module(SCRIPT)
+
+EXPECTED_TICKETS = {5273992, 5273994, 5273997, 5273999, 5274000, 5274329,
+                    5274764, 5274768, 5274772, 5274773, 5274774, 5274775}
+
+
+def main_output(ids):
+    with mock.patch.object(SCRIPT, "all_ids", return_value=ids):
+        f = io.StringIO()
+        with contextlib.redirect_stdout(f):
+            SCRIPT._main()  # pylint: disable=protected-access
+            return f.getvalue()
+
+
+@patch_path_run
+@patch_path_fake_executables("unknown_tickets")
+@patch_getpwuid("resalloc")
+def test_unknown_tickets():
+    subprocess.call("ps")
```

Comment:
Would it be the same to just read the `ps` file and print it to STDOUT? So that we don't have to run a script.

Reply:
This one is a noop and useless debugging thingy - going to drop.  `ps` is executed by the `SCRIPT` itself.

Reply:
Yeees, that makes much more sense now. Thank you.
(We fake the `ps` binary, provide PATH to the faked binary, and the tested script has no idea it was mocked)